tst/regression/run_test.py

```diff
@@ -138,6 +138,11 @@ class TestError(RuntimeError):
                         nargs=1,
                         help='change MPI run wrapper command (e.g. for job schedulers)')
 
+    parser.add_argument('--mpirun_ranks',
+                        default=[],
+                        action='append',
+                        help='add option to specify number of ranks')
+
```

Comment:
Why not add

```
nargs=1
```

Reply:
While in principle I'd be great to enforce this, I'm not sure we can given our logic in the following.
For example, for mpirun the effective line for `run_test.py` reads
`run_test.py --mpirun=mpirun --mpirun_ranks=-n --mpirun_ranks=4 ...`
whereas for jsrun is reads like
`run_test.py --mpirun=jsrun --mpirun_ranks=-a --mpirun_ranks=4 ...`

One option would be to split mpirun_ranks even further and separate the flag (`-n`, `-a`, ...) from the the actual number.

Reply:
I would be in favor of splitting `mpirun_ranks` into `mpirun_rank_flag` and `mpirun_num_ranks` or something.

Reply:
Done now #435

Reply:
Great.